### PR TITLE
sync/db: add just, hyperfine, k9s, and procs

### PR DESCRIPTION
**Pull request by hdhoang**
Parts of #85

i found that procs zip file is not easy to support yet, so i opened a PR there and we will wait for their next release.

### Additional tasks

- [x] Updated CHANGELOG.md

hope this helps!

### CODE REVIEW

**Comment by hdhoang** [comment]
i'm gonna fix default & adopt new procs version too!

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ available [on GitHub][2].
   Adds new error message RepoError::NotFound when fetch_release_info
   returns 404
   (by [@crudiedo][crudiedo])
+* [#147](https://github.com/chshersh/tool-sync/issues/147)
+  Supports casey/just, sharkdp/hyperfine, and dalance/procs natively.
+  However, dalance/procs Windows release is not supported yet.
```
Comment (chshersh) [inline CHANGELOG.md:36]: I see that the new release of `procs` was made recently so this should be possible to support now 👍🏻

**Review by chshersh — COMMENTED** [review]
Thanks for the update so far!

Yes, the list of defaults needs to be updated. README has info about this:

* https://github.com/chshersh/tool-sync#update-the-default-config-golden-test

File: CHANGELOG.md (hunk 1 above)
Comment (hdhoang) [reply CHANGELOG.md]: i updated status, and added k9s too!

**Comment by chshersh** [comment]
@hdhoang Let's also patch the test suite for syncing all tools to make sure they indeed work. Relevant parts:

* https://github.com/chshersh/tool-sync/blob/main/tests/sync-full.toml
* https://github.com/chshersh/tool-sync/blob/38663d0a551ba341e1568261bb1ad92df412a3b3/.github/workflows/ci.yml#L84-L104
* https://github.com/chshersh/tool-sync/blob/38663d0a551ba341e1568261bb1ad92df412a3b3/.github/workflows/ci.yml#L117-L145

**Review by chshersh — APPROVED** [review]
Thanks a lot! 🏆

**Pull request merged**